helpers/getLocalizeResources.js

```diff
@@ -284,26 +284,23 @@ function shouldFetchOverrides() {
 
 function fetchOverride(formatFunc) {
 
-	let url, res;
+	let resource, res;
 
 	if (shouldUseBatchFetch()) {
 
 		// If batching is available, pool requests together.
 
-		url = formatFunc(OverrideLanguage);
-		url = new URL(url).pathname;
-
-		res = fetchWithPooling(url);
+		resource = formatFunc(OverrideLanguage);
```

Comment:
The formatFunc doesn't take any properties, should we remove this? https://github.com/BrightspaceHypermediaComponents/activities/blob/master/components/d2l-activity-editor/mixins/d2l-activity-editor-lang-mixin.js#L9

Reply:
Good catch, yeah I think this is good to remove. I'm actually not sure why it ever took args, maybe the plan was at some point to request per lang instead of one override file. Either way, I think this arg is good to remove.